numpy/polynomial/chebyshev.py

```diff
@@ -800,7 +800,7 @@ def chebdiv(c1, c2):
     # c1, c2 are trimmed copies
     [c1, c2] = pu.as_series([c1, c2])
     if c2[-1] == 0:
-        raise ZeroDivisionError()
+        raise ZeroDivisionError
```

Comment:
The trouble with automated fixes is that one sees things that really should be better, such as here: why is there no helpful message? I guess one can hope to find this again, but seems a bit sad.

Reply:
Could add a `FIXME` comment.

Reply:
Good idea, I'll add FIXME comments.